lib/alephant/broker/load_strategy/s3/sequenced.rb

```diff
@@ -6,11 +6,7 @@ module LoadStrategy
       module S3
         class Sequenced < Base
           def sequence(component_meta)
-            sequencer.get_last_seen component_meta.key
-          end
-
-          def headers(component_meta)
-            { "X-Sequence" => sequence(component_meta).to_s }
+            @sequence ||= sequencer.get_last_seen component_meta.key
```

Comment:
Instance variable no longer needed because `sequence` is now only called one (due to `version` method being removed).